openpower/package/hostboot/p9Patches/hostboot-0004-remove-sbfw-check-hbDistribute.patch

```diff
@@ -0,0 +1,40 @@
+From 52929548d943e0387fabb70d658905120cad7cd8 Mon Sep 17 00:00:00 2001
```

Comment:
Yep. That is what I wanted to make sure of. That a real fix was coming down the line.